### PR TITLE
BERTE-565 handle status check when workflow is pending

### PR DESCRIPTION
**Pull request by tcarmet**

### CODE REVIEW

File: bert_e/git_host/github/__init__.py [hunk 1]
```diff
@@ -508,17 +507,29 @@ def __str__(self) -> str:
...
+    """
+    GET_URL = "/repos/{owner}/{repo}/actions/runs/{id}"
+    CREATE_URL = "/repos/{owner}/{repo}/actions/runs"
+    LIST_URL = CREATE_URL
+    SCHEMA = schema.WorkflowRun
+    LIST_SCHEMA = schema.WorkflowRuns
+
+
+class AggregatedCheckSuites(base.AbstractGitHostObject,
+                            base.AbstractBuildStatus):
```
Comment (tcarmet) [inline bert_e/git_host/github/__init__.py:522]: I noticed that a dependabot check-suite stay in there, can you make sure that we are ignoring it? I don't remember if this was the case with check-runs.

**Review by tcarmet — COMMENTED** [review]
Wish I could approve but it's my own PR. lol

Found a detail to check but LGTM

File: bert_e/git_host/github/__init__.py [hunk 2]
```diff
@@ -528,30 +550,79 @@ def sort_f(check_run):
...
+
+        return AggregatedWorkflowRuns.get(
+            client=self.client,
+            owner=owner, repo=repo, ref=ref,
+            params={
+                'event': 'workflow_dispatch',
+                'branch': self._check_suites[0]['head_branch']
+            })
+
+    def remove_unwanted_workflow_dispatch(self):
```
Comment (tcarmet) [inline bert_e/git_host/github/__init__.py:586]: I would rename this one as it's not only the workflow dispatch
```suggestion
    def remove_unwanted_workflows(self):
```

File: bert_e/git_host/github/__init__.py [hunk 3]
```diff
@@ -528,30 +550,79 @@ def sort_f(check_run):
...
+
+        self._check_suites = list(filter(
+            lambda elem: elem['id'] not in dispatched,
+            self._check_suites
+        ))
+
+        self._check_suites = list(filter(
+            lambda elem: elem['app']['slug'] == 'github-actions',
+            self._check_suites
+        ))
```
Comment (tcarmet) [inline bert_e/git_host/github/__init__.py:609]: :ok_hand:

**Review by erwan-b — APPROVED** [review]

**Review by bertranddemiddelaer — APPROVED** [review]

**Pull request merged**